portfolio-project-matching-app/backend/dao.js

```diff
@@ -76,10 +77,11 @@ const createDoc = async (coll, payload) => {
         let ownerRef;
         if (coll === 'projects') {
             ownerRef = await getDocReferenceById('users', payload.ownerId);
+            payload.owner = ownerRef;
```

Comment:
Not sure how I forgot this when I originally created this function. Pretty important that the owner's user document reference _actually_ gets added to the payload.